src/rest_framework_dso/utils.py

```diff
@@ -0,0 +1,10 @@
+from django.utils.functional import LazyObject, empty
+
+
+def unlazy_object(obj):
```

Comment:
Clever tricks with the lazy object! Are you not afraid the could break, because it uses Django internals?

Reply:
No not really, it's covered by tests and a pretty stable part of Django.